Makefile

```diff
@@ -26,6 +27,10 @@ clean:
 	rm -vrf build/
 .PHONY: clean
 
+lint:
+	golangci-lint run
```

Comment:
prob a dumb q - would this catch anything that gofmt or the compile step wouldn't catch?

Reply:
yes, golangci-lint collects hundreds of lint tools, the compiler does some checks but not style linting

https://golangci-lint.run/usage/linters/